src/compiler/crystal/crystal_path.cr

```diff
@@ -7,7 +7,20 @@ module Crystal
       getter filename
       getter relative_to
 
-      def initialize(@filename : String, @relative_to : String?)
+      def initialize(
+        @filename : String,
+        @relative_to : String?,
+        @entries : Array(String)
```

Comment:
Alternatively, this could potentially be named to be more clear instead of mirroring the outside class
Potentially `search_paths`